bin/startup-start

```diff
@@ -25,7 +26,9 @@ program
   .option('-d, --dev', 'run in development mode')
   .option('--debug', 'run the debugger')
   .option('--debug-brk', 'run the debugger with a break on start')
+  .option('--node-inspector', 'run node-debug to enable debugging in the browser')
   .option('--use-strict', 'use strict mode')
+  .option('--inspect', 'Node 6.3+ Inspector to enable Chrome Dev tools debugging')
```

Comment:
now that i'm looking at this, is this even needed? node inspector works just fine with mocha and gulp according to [the documentation](https://github.com/node-inspector/node-inspector#how-do-i-debug-gulp-tasks). couldn't you just run `node-debug $(which startup-start)`? This would allow you to pass all of the configuration options you need while keeping `startup` small and focused